mona_openai/util/math.py

```diff
@@ -0,0 +1,18 @@
+"""
+Utility module for math-related functionality.
+"""
+
+
+class InvalidQuotientssInputException(Exception):
```

Comment:
Nope. Because it's internal and is actually an error (I should rename this). It's not something that should be raised by the package unless there's a bug.

This also reminds me I should expose all the other exceptions in the init file